src/layer/tile/GridLayer.js

```diff
@@ -543,8 +547,21 @@ export var GridLayer = Layer.extend({
 		return zoom;
 	},
 
+	_usedZoom: function(zoom) {
+		var options = this.options;
+		if (options.usedLevelForFractional === 'round') {
+			return this._clampZoom(Math.round(zoom));
+		}
+		else if (options.usedLevelForFractional === 'ceil') {
```

Comment:
A quick grep shows that Leaflet repo seems to use else ifs/elses on the same line as the previous block's closing brace so you might want to fix it so that it won't prevent taking this in (I need this very same feature myself :-)).